app/talk/discussion-preview.cjsx

```diff
@@ -43,7 +43,7 @@ module.exports = React.createClass
       {owner, name} = @props.params
       "/projects/#{owner}/#{name}/talk/#{discussion.board_id}/#{discussion.id}"
 
-    else if @props.project # otherwise fetch from project
+    else if @props.project.slug # otherwise fetch from project
```

Comment:
This component needs a default project prop since one will not always be passed down from the parent component.